Community/network_manager/network_manager_page.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
+
+# Various Flask framework items.
+import os
+import sys
+from flask import url_for, redirect, render_template, flash, g
+from bluecat import route, util
+import config.default_config as config
+from main_app import app
+from .network_manager_form import GenericFormTemplate
+
+
+def module_path():
+    encoding = sys.getfilesystemencoding()
+    return os.path.dirname(os.path.abspath(str(__file__, encoding)))
```

Comment:
In Python3.x, `str(__file__, encoding)` is no longer supported. Instead of the two lines here, the following should work:

```suggestion
    return os.path.dirname(os.path.abspath(str(__file__)))
```

Also can be seen in this example: https://github.com/bluecatlabs/gateway-workflows/blob/master/Examples/Alias%20Record/delete_alias_record_example/delete_alias_record_example_page.py#L35

Reply:
@chrisjws Can you please make it Python 3 compatible, so it won't crash if 18.10.2 , then we can merge.

Thank you,
Victor